data/static/js/dir_listing.js

```diff
@@ -24,4 +24,19 @@ $('.moviePanel').on('show.bs.collapse', function () {
   var src = $(this).attr('src');
   var type = $(this).attr('type');
   $(this).html("<video controls><source src='" + src + "' type='"+ type +"'></video>");
-})
\ No newline at end of file
+})
+
+$('.dlfiles-div').on('click', function (){
+  var downloadNodes = document.querySelectorAll('.downloadFileLink');
+  for (var i = 0; i < downloadNodes.length; i++) {
+    downloadNodes[i].click();
+  }
+});
+
+function DownloadAllFiles() {
+  var downloadNodes = document.querySelectorAll('.downloadFileLink');
+  for (var i = 0; i < downloadNodes.length; i++) {
+    downloadNodes[i].click();
+  }
+  return downloadNodes;
+}
```

Comment:
Winders didn't add a newline to the end of the file. Plz fix (with 'git commit --amend' followed by a force push).